### PR TITLE
Initial tag logic

### PR DESCRIPTION
**Pull request by ltagliaferri**
First step towards #440 
@sheesh you can implement the tags based on what you see fit following this merge

### CODE REVIEW

File: content/open-source/sigstore/policy-controller/disallowing-unsafe-sysctls-with-policy-controller.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ To follow along with this guide outside of the terminal that is embedded on this
 
 If you are using the terminal that is embedded on this page, then all the prerequsites are installed for you. Note that it may take a minute or two for the Kubernetes cluster to finish provisioning. If you receive any errors while running commands, retry them after waiting a few seconds.
 
-Once you have everything in place you can continue to the first step and confim that the Policy Controller is working as expected.
+Once you have everything in place you can continue to the first step and confirm that the Policy Controller is working as expected.
```
Comment (ltagliaferri) [inline content/open-source/sigstore/policy-controller/disallowing-unsafe-sysctls-with-policy-controller.md:29]: I fixed some policy controller typos

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-rego-policies.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ description: "Writing Rego-based policies for Chainguard Enforce"
 date: 2023-01-12T15:56:52-07:00
 lastmod: 2023-01-12T15:56:52-07:00
 draft: false
-images: []
+images: ["SBOM"]
```
Comment (jamonation) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-rego-policies.md:8]: I think this should be the new tags field?

```suggestion
tags: ["SBOM"]
```

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ description: "Chainguard Enforce User Onboarding"
 date: 2022-07-15T15:22:20+01:00
 lastmod: 2022-12-15T15:22:20+01:00
 draft: false
-images: []
+images: ["SBOM"]
```
Comment (jamonation) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding.md:8]:
```suggestion
tags: ["SBOM"]
```

**Review by jamonation — APPROVED** [review]
LGTM! Minor fixes with some articles with suggestions inline.

I did spot one thing with the OpenVEX page - the description field is incomplete. It reads:
`description: "Using vexctl to "` which shows up like this on the page:

Screenshot 2023-03-09 at 10 21 22 AM



So a small update there and this looks good to go.

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-rego-policies.md (hunk 2 above)
Comment (ltagliaferri) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-rego-policies.md:8]: oh gosh, thanks

**Pull request merged**